sites/fast-website/src/app/components/side-navigation/side-navigation.ts

```diff
@@ -0,0 +1,48 @@
+import { attr, observable, FASTElement } from "@microsoft/fast-element";
+import {
+    CommunityContentPlacementData,
+    communityContentPlacementData,
+} from "../../data/community.data";
+import { throttle } from "lodash";
+
+export class SideNavigation extends FASTElement {
+    @attr
+    public category: string;
+
+    //can be removed to used sectionArray.length once index is restructured to have the correct amount of sections.
+    @attr
+    public sections: number;
+
+    @observable
+    public currentSection: number = 0;
+
+    @observable
+    public sectionArray: HTMLElement[] = Array.from(document.querySelectorAll("section"));
+
+    public updatecurrentSection = (): void => {
+        const fromTop = window.scrollY;
+
+        //replace sections with sectionArray.length after rebase with John's code that reduces sections down to 5
```

Comment:
can you add a TODO here so that this is not missed, eg `// TODO: my comment`

Reply:
I will add that in. Thank you.